chirpstack/Cargo.toml

```diff
@@ -234,10 +234,22 @@
       "/etc/chirpstack/region_us915_6.toml",
       "/etc/chirpstack/region_us915_7.toml",
     ]
+    provides = "chirpstack"
```

Comment:
As now we have a package name `chirpstack` I think this is no longer required? E.g. for the editor example you gave, `editor` is a virtual package provided by `nano`, `vim` etc... but there is no exact package called `editor`. When running `apt install chirpstack` it is no longer expected that `apt` will return a list of packages that provide `chirpstack` as we expect `apt` to just install `chirpstack`.

Reply:
That's right, let me remove that.